### PR TITLE
[Feature] Posec3d demo

### PR DESCRIPTION
**Pull request by kennymckormick**

### CODE REVIEW

File: demo/demo_posec3d.py [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    return args
+
+
+def frame_extraction(video_path, short_side):
+    """Extract frames given video_path.
+
+    Args:
+        video_path (str): The video_path.
+    """
+    # Load the video, extract frames into /tmp/video_name
```
Comment (dreamerlin) [inline demo/demo_posec3d.py:126]: ./tmp/video_name

File: demo/demo_posec3d.py [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    pose_model = init_pose_model(args.pose_config, args.pose_checkpoint,
+                                 args.device)
+    vis_frames = [
+        vis_pose_result(pose_model, frame_paths[i], pose_results[i])
+        for i in range(num_frame)
+    ]
+    _ = [
+        cv2.putText(frame, action_label, (10, 30), FONTFACE, FONTSCALE,
+                    FONTCOLOR, THICKNESS, LINETYPE) for frame in vis_frames
+    ]
```
Comment (dreamerlin) [inline demo/demo_posec3d.py:261]:
```python
for frame in vis_frames:
    cv2.putText(frame, action_label, (10, 30), FONTFACE, FONTSCALE, FONTCOLOR, THICKNESS, LINETYPE)
```

File: demo/demo_posec3d.py [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+import os.path as osp
+import shutil
+
+import cv2
+import mmcv
+import numpy as np
+import torch
+from mmcv import DictAction
+from mmcv.runner import load_checkpoint
+from tqdm import tqdm
```
Comment (dreamerlin) [inline demo/demo_posec3d.py:12]: Can we use `mmcv.ProgressBar` instead?

**Comment by gengenkai** [comment]
Has tested the script demo/demo_posec3d.py

**Review by gengenkai — APPROVED** [review]

**Pull request merged**